src/Infrastructure.Dapper/AdminConsole/Repositories/OrganizationUserRepository.cs

```diff
@@ -520,4 +522,32 @@ public async Task<IEnumerable<OrganizationUserPolicyDetails>> GetByUserIdWithPol
             return results.ToList();
         }
     }
+
+    /// <inheritdoc />
+    public UpdateEncryptedDataForKeyRotation UpdateForKeyRotation(
+        Guid userId, IEnumerable<OrganizationUser> resetPasswordKeys)
+    {
+        return async (SqlConnection connection, SqlTransaction transaction) =>
+        {
+            const string sql = @"
```

Comment:
What is the reasoning for having this hardcoded? Usually we created a stored procedure

Reply:
The other domains didn't have a TVPs already in the database, which made bulk updates a little more difficult with store procedures (TVPs just mirror the model and must be maintained, which I didn't want force on each team).

In order to accomplish the bulk updates I wanted, I ended up keeping the old ADO.NET and the associated sql scripts.

While it is easier to change the SQL when its hardcoded like this, I understand if you want me to create a new Stored Procedure (it would just increase the scope of this even more 😅)

Reply:
That makes complete sense to me, I didn't think about the TVPs. 
I'm not opposed at all to have the queries hardcoded (it is easier to maintain!) but we need a strong reason to diverge from the usual pattern.